variants/esp32-spiram/README.md

```diff
@@ -0,0 +1,5 @@
+# ESP32-SPIRAM
+
+A variant of the ESP32 envelope that supports SPIRAM (external memory).
+Typically, these are VROOM modules with 4MB of SPIRAM, but some modules
```

Comment:
```suggestion
Typically, these are WROOM modules with 4MB of SPIRAM, but some modules
```

Reply:
actually should have been WROVER.
done.